bears/python/PycodestyleBear.py

```diff
@@ -37,8 +37,10 @@ def create_arguments(
         """
         arguments = [r'--format=%(row)d %(col)d %(code)s %(text)s']
 
+        ignore_values = 'E121, E123, E126, E133, E226, E241, E242, E704, W503'
```

Comment:
You mean to put it in parameters of create_arguments ?
If I understood it right, putting default values in param itself will not resolve the issue. Because we need to append the default values to param values, and using param only will replace it.